const { catchAsyncError } = require("../utils/catchAsyncError");

exports.markStatus = catchAsyncError(async (req, res, next) => {});
exports.changeStatus = catchAsyncError(async (req, res, next) => {});
